tests/testthat/test-survival-bayes.R

```diff
@@ -0,0 +1,301 @@
+
+test_that("survival analysis - tuning via Bayesian search", {
+  skip_on_cran()
+  skip_if_not_installed("modeldata")
+  skip_if_not_installed("censored")
+
+  suppressPackageStartupMessages(library(dplyr))
+  suppressPackageStartupMessages(library(censored))
+  suppressPackageStartupMessages(library(modeldata))
+  suppressPackageStartupMessages(library(rsample))
+  suppressPackageStartupMessages(library(yardstick))
+  suppressPackageStartupMessages(library(recipes))
+  suppressPackageStartupMessages(library(workflows))
+  suppressPackageStartupMessages(library(dials))
+
+  # ----------------------------------------------------------------------------
+
+  data("mlc_churn")
+
+  mlc_churn <-
+    mlc_churn %>%
+    mutate(
+      churned = ifelse(churn == "yes", 1, 0),
+      event_time = Surv(account_length, churned)
+    ) %>%
+    select(event_time, account_length, voice_mail_plan) %>%
+    slice(1:500)
+
+  set.seed(6941)
+  churn_split <- initial_split(mlc_churn, prop = 4/5)
+  churn_tr <- training(churn_split)
+  churn_te <- testing(churn_split)
+  churn_rs <- bootstraps(churn_tr, times = 2)
+
+  # ----------------------------------------------------------------------------
+
+  eval_times <- c(100, 10, 150)
+  event_metrics <- metric_set(brier_survival, brier_survival_integrated,
+                              concordance_survival, roc_auc_survival)
+  sr_tune_spec <- survival_reg(dist = tune())
+  poly_rec <- recipe(event_time ~ ., data = churn_tr) %>%
+    step_poly(account_length, degree = tune())
+  sr_tune_wflow <-
+    workflow() %>%
+    add_model(sr_tune_spec) %>%
+    add_recipe(poly_rec)
+
+  sr_tune_param <-
+    sr_tune_wflow %>%
+    extract_parameter_set_dials() %>%
+    update(
+      degree = degree(c(1, 10)),
+      dist = surv_dist(c(c("loglogistic", "lognormal")))
+    )
+
+  # ------------------------------------------------------------------------------
+
+  set.seed(1)
+  expect_snapshot(
```

Comment:
Why are you putting this into a snapshot? I'd put it in a `suppressMessages()` unless you want to specifically test the output.